src/test/scala/org/broadinstitute/dsde/firecloud/service/NihApiServiceSpec.scala

```diff
@@ -89,7 +88,7 @@ class NihApiServiceSpec extends ApiServiceSpec {
     Post("/sync_whitelist") ~> sealRoute(services.syncRoute) ~> check {
       status should equal(NoContent)
     }
-    assert(services.rawlsDao.groups(dbGapAuthorizedUsersGroupName).equals(Set(services.thurloeDao.linkedUserId)))
+    assert(services.rawlsDao.groups(dbGapAuthorizedUsersGroupName).contains(services.thurloeDao.linkedUserId))
```

Comment:
these tests check to see if the user is in the dbGapAuthorizedUsersGroupName group, what about target?